architecture/adr-list/adr-008-ledger-resources.md

```diff
@@ -230,140 +190,217 @@ Example using the Veramo CLI:
 
 ```jsonc
-{ "collectionId": "DAzMQo4MDMxCjgwM" }
+{ "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4" }
 ```
 
-#### QueryGetCollectionResourcesResponse
+#### QueryCollectionResourcesResponse
 
-* Resources: [ResourceHeader\[\]](#resourceheader)
+* Returns `collection of resources` created by the specific `collection ID`.
 
 Example:
 
 ```jsonc
-{ "resources":  [<ResourceHeader1>, <ResourceHeader2>] }
+{
+    "resources": [
+        {
+            "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+            "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+            "name": "TicketScheme",
+            "version": "1.0",
+            "resourceType": "CL-Schema",
+            "alsoKnownAs": [
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": ""
+                }
+            ]
+        },
+        {
+            "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+            "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+            "name": "PassportScheme",
+            "version": "1.0",
+            "resourceType": "CL-Schema",
+            "alsoKnownAs": [
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": ""
+                },
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": "Alternative URI description"
+                }
+            ]
+        }
+    ]
+}
 ```
 
-#### QueryGetResourceRequest
+#### QueryResourceRequest
 
-* Collection ID: String - an identifier of linked DIDDoc
-* ID: String - unique resource id
+* `Collection ID`: String - an identifier of linked DIDDoc
+* `ID`: String - unique resource id
 
 Example:
 
 ```jsonc
 { 
-  "collectionId": "DAzMQo4MDMxCjgwM",
-  "id": "bb2118f3-5e55-4510-b420-33ef9e1726d2"
+  "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+  "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f"
 }
 ```
 
-#### QueryGetResourceResponse
+#### QueryResourceResponse
 
-* Resource: [Resource](#resource)
+* Returns `Resource` with a given `collection ID` and `ID`.
 
 Example:
 
 ```jsonc
-{ "resource":  <Resource> }
+{
+    "resource": {
+        "resource": {
+            "data": "eyJ0ZXN0IjogInRlc3QifQ=="
+        },
+        "metadata": {
+            "collectionId": "91e5f0cf-5f1e-5c19-97d3-d313e84033b4",
+            "id": "54cb8b4d-af33-4606-bc54-0f035ee30e0f",
+            "name": "PassportScheme",
+            "version": "1.0",
+            "resourceType": "CL-Schema",
+            "alsoKnownAs": [
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": ""
+                },
+                {
+                    "uri": "https://example.com/alternative-uri",
+                    "description": "Alternative URI description"
+                }
+            ],
+            "media_type": "application/json",
```

Comment:
let's make all outputs from ledger in snake case.
Also, please add here `next_version_id` and `previous_version_id`

Reply:
Done! ✅